types/tests/server-test.ts

```diff
@@ -0,0 +1,63 @@
+import { Response, Server } from "miragejs";
+
+export default function config(this: Server): void {
+  this.namespace = "foo";
+  this.urlPrefix = "/api";
+  this.timing = 123;
+  this.logging = true;
+
+  this.get("/foo");
+  this.put("/foo");
+  this.post("/foo");
+  this.patch("/foo");
+  this.options("/foo");
+  this.del("/foo");
+
+  this.passthrough("/_coverage/upload");
+  this.passthrough(request => request.queryParams.skipMirage);
+
+  this.loadFixtures();
+  this.seeds(this);
+  this.routes();
+
+  this.shutdown();
```

Comment:
As above, I suggest we add tests for the expect result type for each of these.

Reply:
Thanks. Assertions added. ;)